tests/python/unittest/test_init.py

```diff
@@ -67,10 +67,23 @@ def test_bilinear_init():
     bili_1d = np.array([[1/float(4), 3/float(4), 3/float(4), 1/float(4)]])
     bili_2d = bili_1d * np.transpose(bili_1d)
     assert (bili_2d == bili_weight.asnumpy()).all()
-    
+
+def test_const_init_dumps():
+    # test NDArray input
+    init = mx.init.Constant(mx.nd.ones(5))
```

Comment:
it might be good if we could randomize the shape and dimension for this test and test it for different dimensions

Reply:
Added randomized shape and dimension for the array in test, it can take values between 1 and 10 with 1D to 5D shapes.